source/includes/azure/_instances.md

```diff
@@ -314,4 +314,47 @@ curl -X POST \
 
  <code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id?operation=stop</code>
 
- Stop an existing instance. The instance must be in either the power state *PowerState/running* or *PowerState/starting* status for this operation to work. 
+ Stop an existing instance. The instance must be in either the power state *PowerState/running* or *PowerState/starting* status for this operation to work.
+
+
+<!-------------------- ATTACH A DISK -------------------->
+
+#### Attach a disk
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/azure-conn/test_env/instances/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/vm1?operation=attach"
+
+# Request should look like this
+```
+```shell
+# Request Example:
+```
+
+```json
+{
+  "diskToAttach" : "/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/disks/disktest1",
+  "diskLun": "10"
+}
+```
+
+
+```shell
+# Response Example
+```
+```json
+{
+  "taskId": "802c5ae0-8431-47dc-9abe-7e10a9badddc",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id?operation=attach_disk</code>
+
+
+Required | &nbsp;
+------ | -----------
+Disk ID: `diskToAttach` <br/>*String* | The Id of the disk that you want to be attached to the instance.
+LUN: `diskLun`<br/>*Integer* | The logic unit number of a disk. Can only be between 1 and 63 inclusive.
```

Comment:
Same here, no need for the Disk ID: or LUN:

Reply:
removed it here ab36c1c83d0980c67fd84a0f0cafeaace182f146